lib/modules/platform/gitlab/readme.md

```diff
@@ -44,6 +48,8 @@ If you're using a private [GitLab container registry](https://docs.gitlab.com/ee
 - Make sure the user that owns the access token is a member of the corresponding GitLab projects/groups with the right permissions.
 - Make sure the access token has the `read_registry` scope.
 
+You may also use a dedicated [Deploy Token](https://docs.gitlab.com/ee/user/project/deploy_tokens/) instead of using `RENOVATE_TOKEN` as the password in the above host rule example.
```

Comment:
Thanks for clarifying this. We indeed use a PAT as `RENOVATE_TOKEN` and a deploy token for `DOCKER_REGISTRY_GITLAB_COM_*` and `PACKAGIST_GITLAB_COM_*`-